apps/ledger-live-mobile/src/screens/Onboarding/steps/deviceSelection.tsx

```diff
@@ -18,28 +18,32 @@ import {
 import nanoSSvg from "../assets/nanoS";
 import nanoSPSvg from "../assets/nanoSP";
 import nanoXSvg from "../assets/nanoX";
-import DiscoverCard from "../../Discover/DiscoverCard";
 import DeviceSetupView from "../../../components/DeviceSetupView";
 import { RootStackParamList } from "../../../components/RootNavigator/types/RootNavigator";
 import { NavigateInput } from "../../../components/RootNavigator/types/BaseNavigator";
+import ChoiceCard from "./ChoiceCard";
 
 const nanoX = {
   SvgDevice: nanoXSvg,
+  source: require("../../../../assets/images/devices/NanoXCropped.png"),
```

Comment:
So we don't use the SVG now ? If we only use the png source, we should remove the `SvgDevice` and the import

Reply:
Are we sure we don't want to use SVG anymore though ?